linux_os/guide/system/permissions/files/permissions_var_log_dir/file_permissions_var_log_gdm/tests/lenient_permissions.fail.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p /var/log/gdm
```

Comment:
we should have a test for stricter permissions too.

Reply:
Fixed